src/core/reactor.cc

```diff
@@ -2545,7 +2545,9 @@ uint64_t
 reactor::pending_task_count() const {
     uint64_t ret = 0;
     for (auto&& tq : _task_queues) {
-        ret += tq->_q.size();
+        if(tq) {
```

Comment:
Coding style: space after `if`.

Reply:
@avikivity the space has been added, the branch is updated